package.json

```diff
@@ -45,14 +46,17 @@
     "log-update": "^1.0.2",
     "node-emoji": "^1.0.4",
     "noop-fn": "^1.0.0",
+    "nsp": "^2.1.0",
     "os-family": "^1.0.0",
     "pinkie-promise": "^1.0.0",
-    "spawn-cmd": "0.0.2"
+    "spawn-cmd": "0.0.2",
+    "uglify-js": "^2.5.0"
```

Comment:
right, those shouldn't be there,  they had known vulnerabilities so i was testing them,  forgot to take them out.